bukkit/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'java'
-    id 'com.github.johnrengelman.shadow' version '8.1.1'
+    id 'io.github.goooler.shadow' version "8.1.8"
```

Comment:
I've switched to this fork to support java version 21, not sure if this is also necessary for java 17.